SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -1087,7 +1087,7 @@ void CEulerSolver::SetNondimensionalization(CConfig *config, unsigned short iMes
   su2double Beta          = config->GetAoS()*PI_NUMBER/180.0;
   su2double Mach          = config->GetMach();
   su2double Reynolds      = config->GetReynolds();
-  bool unsteady           = (config->GetTime_Marching() != NO);
+  bool unsteady           = (config->GetTime_Marching() != TIME_MARCHING::STEADY);
```

Comment:
using another enum's value like NO in this case. Note to self: Maybe introduce an `inline bool isdual_time` and `isunsteady` in option_structure as it is used quite often